frame/contracts/src/wasm/runtime.rs

```diff
@@ -1741,6 +1748,20 @@ pub mod env {
 		Ok(ctx.ext.caller_is_origin() as u32)
 	}
 
+	/// Checks whether the caller of the current contract is root.
+	///
+	/// Note that this is only possible when root is the origin of the whole call stack.
+	///
+	/// A return value of `true` indicates that this contract is being called by a root origin
+	/// and `false` indicates that the caller is a signed origin.
+	///
+	/// Returned value is a `u32`-encoded boolean: (`0 = false`, `1 = true`).
+	#[prefixed_alias]
```

Comment:
should this be marked as #[unstable] ?

also you can drop the prefixed_alias, this is for legacy runtime functions
https://github.com/paritytech/substrate/pull/13724#discussion_r1158725288

Reply:
Thanks for the `prefixed_alias` thing.
Not sure about `unstable` though, @athei what do you say?

Reply:
From Alex: _They should always start out as unstable. We stabilize once they supported in ink! and we have a integration test for them._
So adding #[unstable]